api/proto/buf.lock

```diff
@@ -4,8 +4,10 @@ deps:
   - remote: buf.build
     owner: gogo
     repository: protobuf
-    commit: b03c65ea87cdc3521ede29f62fe3ce239267c1bc
```

Comment:
Buf v1.32.0 is really unhappy about this hash. I ended up deleting the lock files and creating them again with `buf dep update`.

AFAICT this is still the same underlying commit, except the hash is from the Buf registry: https://buf.build/gogo/protobuf/commits/commit/4df00b267f944190a229ce3695781e99. If you look under "associated labels" you can see b03c65ea87cdc3521ede29f62fe3ce239267c1bc in there.